libbeat/processors/add_cloud_metadata/provider_aws_ec2_test.go

```diff
@@ -82,7 +82,7 @@ func TestRetrieveAWSMetadata(t *testing.T) {
 
 	expected := common.MapStr{
 		"cloud": common.MapStr{
-			"provider": "ec2",
+			"provider": "aws",
```

Comment:
I wonder if this is also going to break some of our dashboards?

As I would consider this a bug in 7 I'm good with getting this in even though it's a breaking change.